graphql/resolver.go

```diff
@@ -35,6 +36,52 @@ func New(apiURL string) Config {
 			sc: dbConnector,
 		},
 	}
+	c.Directives.RequireHostAccess = func(ctx context.Context, obj any, next graphql.Resolver, access HostAccessLevel) (any, error) {
+		args, isStringMap := obj.(map[string]interface{})
+		if !isStringMap {
+			return nil, ResourceNotFound.Send(ctx, "host not specified")
+		}
+		hostId, hasHostId := args["hostId"].(string)
+		hostIdsInterface, hasHostIds := args["hostIds"].([]interface{})
+		if !hasHostId && !hasHostIds {
+			return nil, ResourceNotFound.Send(ctx, "host not specified")
+		}
+
+		hostIdsToCheck := []string{hostId}
+		if hasHostIds {
+			for _, v := range hostIdsInterface {
+				hostIdsToCheck = append(hostIdsToCheck, v.(string))
+			}
+		}
+		var requiredLevel int
+		if access == HostAccessLevelEdit {
+			requiredLevel = evergreen.HostsEdit.Value
+		} else {
+			requiredLevel = evergreen.HostsView.Value
+		}
+		user := mustHaveUser(ctx)
+		hostsToCheck, err := host.Find(ctx, host.ByIds(hostIdsToCheck))
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting hosts: %s", err.Error()))
```

Comment:
```suggestion
			return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting hosts: %s", err.Error()))
```
to adhere to the [go conventions](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit?tab=t.0#heading=h.sp5f9pecb7lw).